ui/tests/acceptance/dashboard-test.js

```diff
@@ -53,4 +61,296 @@ module('Acceptance | landing page dashboard', function (hooks) {
+          disable_sealwrap: false,
+          disable_sentinel_trace: false,
+          enable_response_header_hostname: false,
+          enable_response_header_raft_node_id: false,
+          enable_ui: true,
+          experiments: null,
+          introspection_endpoint: false,
+          listeners: [
+            {
+              config: {
+                address: '0.0.0.0:8200',
+                cluster_address: '0.0.0.0:8201',
+                tls_disable: false,
+              },
+              type: 'tcp',
+            },
+          ],
+          log_format: '',
+          log_level: 'debug',
+          log_requests_level: '',
+          max_lease_ttl: '48h',
+          pid_file: '',
+          plugin_directory: '',
+          plugin_file_permissions: 0,
+          plugin_file_uid: 0,
+          raw_storage_endpoint: true,
+          seals: [
+            {
+              disabled: false,
+              type: 'shamir',
+            },
+          ],
+          storage: {
+            cluster_addr: 'https://127.0.0.1:8201',
+            disable_clustering: false,
+            raft: {
+              max_entry_size: '',
+            },
+            redirect_addr: 'http://127.0.0.1:8200',
+            type: 'raft',
+          },
+          telemetry: {
+            add_lease_metrics_namespace_labels: false,
+            circonus_api_app: '',
+            circonus_api_token: '',
+            circonus_api_url: '',
+            circonus_broker_id: '',
+            circonus_broker_select_tag: '',
+            circonus_check_display_name: '',
+            circonus_check_force_metric_activation: '',
+            circonus_check_id: '',
+            circonus_check_instance_id: '',
+            circonus_check_search_tag: '',
+            circonus_check_tags: '',
+            circonus_submission_interval: '',
+            circonus_submission_url: '',
+            disable_hostname: true,
+            dogstatsd_addr: '',
+            dogstatsd_tags: null,
+            lease_metrics_epsilon: 3600000000000,
+            maximum_gauge_cardinality: 500,
+            metrics_prefix: '',
+            num_lease_metrics_buckets: 168,
+            prometheus_retention_time: 86400000000000,
+            stackdriver_debug_logs: false,
+            stackdriver_location: '',
+            stackdriver_namespace: '',
+            stackdriver_project_id: '',
+            statsd_address: '',
+            statsite_address: '',
+            usage_gauge_period: 5000000000,
+          },
+        },
+        wrap_info: null,
+        warnings: null,
+        auth: null,
+      }));
+      await visit('/vault/dashboard');
+      assert.dom(VAULT_CONFIGURATION_SELECTORS.tlsDisable).hasText('Disabled');
+    });
+    test('shows the tls disabled if there is no tlsDisabled returned from server', async function (assert) {
+      this.server.get('sys/config/state/sanitized', () => ({
+        data: {
+          api_addr: 'http://127.0.0.1:8200',
```

Comment:
It might be worth saving this response in the `beforeEach` hook as something like `this.data` and just change the value of the param you're testing in each test

Reply:
i love that idea 💖